proto/sequencerapis/astria/sequencer/v1alpha1/service.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package astria.sequencer.v1alpha1;
+
+import "astria/sequencer/v1alpha1/block.proto";
+
+message GetSequencerBlockRequest {
```

Comment:
I wonder if a nice API would be:

```proto
message GetSequencerBlockRequest {
    int64 height = 1; // or uint32
    optional bytes rollup_id = 2;
}
```

This means you can either get the full block if you leave the field unset. Or you can get the block with the rollup specific transactions filtered out.

Bonus: `optional` and `repeated` are compatible and not a breaking change if we decide to `optional -> repeated` later

Reply:
My general view is to align with standard resource based RPC type architectures there should be a standard `Get` accessor, such that we could map with an http proxy to an http `GET` request with no body or params only an id. 

The other method I would consider "custom". This loosely aligns with practices suggested here: https://google.aip.dev/131, https://google.aip.dev/136